app/src/main/java/org/p2p/wallet/common/mvp/BaseMvpBottomSheet.kt

```diff
@@ -103,12 +103,12 @@ abstract class BaseMvpBottomSheet<V : MvpView, P : MvpPresenter<V>>(
 
     override fun showUiKitSnackBar(message: String, onDismissed: () -> Unit) {
         val root = requireActivity().findViewById<ViewGroup>(android.R.id.content)
-        root.showSnackbarShort(message)
+        root.showSnackbarShort(message, onDismissed)
```

Comment:
oh no, it's conflicts time, do you mind reverting this?

Reply:
what time?